bin/deploy-dev.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -ex
+
+CONTAINER_USER="sail"
+COMPOSE_CMD="/usr/local/bin/docker-compose -f docker-compose.cloud.yml"
+EXEC_CMD="$COMPOSE_CMD exec -T --user $CONTAINER_USER app"
+
+# Build image and (re)create containers
+$COMPOSE_CMD up -d --force-recreate --build --pull always
+
+# Deploy app
+$EXEC_CMD npm install
+$EXEC_CMD composer install --optimize-autoloader
+$EXEC_CMD ./artisan migrate:fresh --force
```

Comment:
I think you'll need to run the seeders too. I usually do this [locally](https://github.com/fluid-project/lcil/blob/dev/README.md#setup-for-developing-locally) by calling `php artisan migrate:fresh --seed`. Which uses the [DatabaseSeeder](https://github.com/fluid-project/lcil/blob/dev/database/seeders/DatabaseSeeder.php). That will add a the constant data (measures), a demo user, and pre-load some sample data into the database. For production, we'd just run the [ConstantsSeeder](https://github.com/fluid-project/lcil/blob/dev/database/seeders/ConstantsSeeder.php) for now.

Reply:
I think the artisan storage:link command may also need to be run.